pysimplesoap/__init__.py

```diff
@@ -12,5 +12,8 @@
 
 TIMEOUT = 60
 
-
-from . import client, server, simplexml, transport
+# to prevent errors during installation
```

Comment:
The library handles these situations like this (using the pycurl as an example) --

``` python
#
# pycurl support.
# experimental: pycurl seems faster + better proxy support (NTLM) + ssl features
#
try:
    import pycurl
except ImportError:
    pass
else:
    try:
        from cStringIO import StringIO
    except ImportError:
        try:
            from StringIO import StringIO
        except ImportError:
            from io import StringIO

    class pycurlTransport(TransportBase):
```